RecoBTag/Combined/python/combinedMVAV2Computer_cfi.py

```diff
@@ -0,0 +1,12 @@
+import FWCore.ParameterSet.Config as cms
+
+combinedMVAV2Computer = cms.ESProducer("CombinedMVAV2JetTagESProducer",
+	jetTagComputers = cms.vstring(
+		'jetProbabilityComputer',
+		'jetBProbabilityComputer',
+		'combinedSecondaryVertexV2Computer',
+		'softPFMuonComputer',
+		'softPFElectronComputer'
+	),
+	weightFile = cms.FileInPath('RecoBTag/Combined/data/CombinedMVAV2_13_07_2015.weights.xml.gz')
```

Comment:
apparently I forgot to commit some files, I indeed implemented switches just as in MuonTagger. just a moment...